### PR TITLE
Resolved pizza list page

### PR DESCRIPTION
**Pull request by Shanmugarmd**
Resolved pizza list page crash & addToCart API added

### CODE REVIEW

File: pizzastore/src/main/java/com/springlabs/pizzastore/domain/PizzaVariant.java [hunk 1]
```diff
@@ -46,14 +44,14 @@
...
+	
 	@ManyToOne
 	@JoinColumn(name = "pizza_id")
-	private Pizza pizzaId;
+	private Pizza pizza;
 
 	@OneToMany
-	@JoinColumn(name = "pizza_variety_id")
-	private List<PizzaVariety> pizzaVarieties;
+	@JoinColumn(name = "id_pizza_variety")
```
Comment (john77eipe) [inline pizzastore/src/main/java/com/springlabs/pizzastore/domain/PizzaVariant.java:53]: Isn't the column name id (from the pizza variety table) also use plural if the object is list/array so use pizzaVarieties

File: pizzastore/src/main/java/com/springlabs/pizzastore/domain/PizzaVariant.java [hunk 2]
```diff
@@ -119,35 +117,39 @@ public void setOnsale(Integer onsale) {
...
-
-	public void setPizzaVarieties(List<PizzaVariety> pizzaVarieties) {
-		this.pizzaVarieties = pizzaVarieties;
+	
+	public void setPizza(Pizza pizza) {
+		this.pizza = pizza;
 	}
 
 	@Override
 	public String toString() {
```
Comment (john77eipe) [inline pizzastore/src/main/java/com/springlabs/pizzastore/domain/PizzaVariant.java:129]: Use StringJoiner for toString

**Review by john77eipe — COMMENTED** [review]
Please see my comments

File: pizzastore/src/main/java/com/springlabs/pizzastore/domain/PizzaVariant.java (hunk 2 above)
Comment (Shanmugarmd) [reply pizzastore/src/main/java/com/springlabs/pizzastore/domain/PizzaVariant.java:129]: ok

**Review by john77eipe — APPROVED** [review]

**Pull request merged**